docs/src/components/components/action/README.md

```diff
@@ -0,0 +1,11 @@
+
+# Action
+
+The **action** component is something to use when you want the user to take an
+action. Buttons should be used for changing the state of a page or transforming
+data. Links should be used for navigation.
+
+## Use
+
+To use the **action** component, add the class `action` to a button or add the
```

Comment:
Worth being specific that this needs to go on a `<button>` element — this is _almost_ specific enough! :)